build/Prepare.targets

```diff
@@ -71,7 +71,8 @@
 
   <Target Name="SetupRestoreSrcPackagesInputsOutputs">
     <ItemGroup>
-      <RestoreSrcPackagesInput Include="$(RepoRoot)/src/**/*.csproj" Exclude="$(RepoRoot)/src/**/%24projectName%24.csproj"/>
+      <RestoreSrcPackagesInput Include="$(RepoRoot)/src/**/*.csproj" />
+      <RestoreSrcPackagesInput Remove="%(RestoreSrcPackagesInput.Identity)" Condition="'%(RestoreSrcPackagesInput.Filename)' == 'Microsoft.DotNet.MSBuildSdkResolver' and '$(IncludeMSBuildSdkResolver)' != 'true'" />
```

Comment:
This project does not exist in the SDK. Actually, no src exists in core-sdk at the moment.

Reply:
Let's handle that as stuff to delete. This whole target is unneeded. cc @johnbeisner 

I think as long as things are still undeleted, then we can take clean merges if they're green. Once it gets deleted, we'll get conflicts and can resolve in favor of the deletion.